pandas/io/parsers/python_parser.py

```diff
@@ -434,6 +452,7 @@ def _infer_columns(
                         if i not in this_unnamed_cols
                     ] + this_unnamed_cols
 
+                    # TODO: Use pandas.io.common.dedup_names instead (see #50371)
```

Comment:
@datapythonista This is one `TODO` you mentioned? Just to be sure and not doing rework? #50371

Reply:
Yes, correct.